service/spi/src/main/java/org/kaazing/gateway/service/ServiceContext.java

```diff
@@ -75,9 +79,9 @@
 
     File getTempDirectory();
 
-    void init() throws Exception;
+    void init() ;
 
-    void start() throws Exception;
+    void start() throws MalformedURLException, RemoteException, URISyntaxException, IOException;
```

Comment:
same feedback as above. Also you will break enterprise.gateway or itests if they have a dependency on this file